bgpd/bgp_evpn_vty.c

```diff
@@ -1887,6 +1887,13 @@ static struct bgpevpn *evpn_create_update_vni(struct bgp *bgp, vni_t vni)
 
 	vpn = bgp_evpn_lookup_vni(bgp, vni);
 	if (!vpn) {
+		/* Check if this L2VNI is already configured as L3VNI */
+		if (bgp_evpn_lookup_l3vni_l2vni_table(vni)) {
+			zlog_err("%u: Failed to create L2VNI %u, it is configured as L3VNI",
```

Comment:
Should this be flog_err ?

Reply:
Done. At the time of this PR submitted flog_err was not there, I have updated to latest this branch and converted to flog_err. Please check the latest changes on this PR.